bubble.rb

```diff
@@ -0,0 +1,49 @@
+def bubble_sort(arr)
+  i = 0
+  j = 0
+  while i < arr.size
+    j = i + 1
+    while j <= arr.size - 1
+      if arr[i] > arr[j]
+        temp = arr[i]
+        arr[i] = arr[j]
+        arr[j] = temp
+      end
+      j += 1
+    end
+    i += 1
+  end
+end
```

Comment:
* [ ] bubble_sort method is implemented and working ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_bubble_sort/sort.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-1))

- Your method is returning `nil` instead of the sorted array.

```ruby
arr = [54, 14, 90, 2, 25]
puts bubble_sort(arr)

>> nil
```